### PR TITLE
Added info about EC2 Reserved Instance Types

### PR DESCRIPTION
**Pull request by lynnlangit**
also added links about 'selling' back via the AWS Reserved Instance Marketplace.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1616,6 +1616,7 @@ Billing and Cost Management
...
 	-	Consider using Reserved Instances when you can predict your longer-term compute needs and need a stronger guarantee of compute availability and continuity than the (typically cheaper) Spot market can provide. However be aware that if your architecture changes your computing needs may change as well so long term contracts can seem attractive but may turn out to be cumbersome.
+	-       There are two types of Reserved Instances - [Standard or Convertible](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/reserved-instances-types.html).  If you purchase excess Standard Reserved Instances, you may offer to 'sell back' unused Reserved Instances via the AWS [Reserved Instance Marketplace](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ri-market-selling-guide.html), this allows you to potential recoup unused, but purchased EC2 compute instance hours by selling them to other AWS customers.
```
Comment (ThanosBaskous) [inline README.md]: Non-blocking nit: 1 space before new sentence.

Comment (ThanosBaskous) [inline README.md]: Non-blocking nit: ‘sell back’ instead of 'sell back'

Comment (ThanosBaskous) [inline README.md]: Non-blocking phrasing suggestion: "Standard or Convertible" -> "Standard and Convertible"

Comment (ThanosBaskous) [inline README.md]: Non-blocking brevity suggestion: "AWS [Reserved Instance Marketplace]" - "AWS" not necessary.

Comment (ThanosBaskous) [inline README.md]: Typo: "potential" -> "potentially"

Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "recoup unused, but purchased EC2 compute instance hours by selling them to other AWS customers" -> "recoup the cost of unused EC2 compute instance hours by selling them to other AWS customers"

**Comment by lynnlangit** [comment]
done

File: README.md [hunk 2]
```diff
@@ -1616,6 +1616,7 @@ Billing and Cost Management
...
 	-	Consider using Reserved Instances when you can predict your longer-term compute needs and need a stronger guarantee of compute availability and continuity than the (typically cheaper) Spot market can provide. However be aware that if your architecture changes your computing needs may change as well so long term contracts can seem attractive but may turn out to be cumbersome.
+	-       There are two types of Reserved Instances - [Standard and Convertible](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/reserved-instances-types.html). If you purchase excess Standard Reserved Instances, you may offer to “sell back” unused Reserved Instances via the [Reserved Instance Marketplace](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ri-market-selling-guide.html), this allows you to potentially recoup the cost of unused EC2 compute instance hours by selling them to other AWS customers.
```
Comment (ThanosBaskous) [inline README.md]: Non-blocking nit (no need to fix before merge): use em dash.

**Comment by lynnlangit** [comment]
nothing else for me to do here, right?

**Comment by ThanosBaskous** [comment]
@lynnlangit not from my end - we aren't requiring fixes for small nitpicky style details anymore (like using em dash) - I try to put "non-blocking" in the description to show this.

I have handed this PR off to @max-zanko for topic review - it is possible that he'll request "content-specific" changes.

**Comment by ThanosBaskous** [comment]
Thanks @lynnlangit!

**Pull request merged**